conversations/common_actions.py

```diff
@@ -0,0 +1,202 @@
+import uuid
+
+from django.db import transaction
+from django.utils import timezone
+
+from .models import Thread, Message
+
+
+class ConversationException(Exception):
+    pass
+
+
+class ConversationValidationError(ConversationException):
+    pass
+
+
+def notify(user, message):
+    """
+    Placeholder for notifications
+    """
+    pass
+
+
+def mark_thread_as_completed(thread, user):
+    # if thread.status not in [
```

Comment:
What about writing this commented out code in the form of a test (a skipped one) so that we don't have commented out implementation without a context